Jenkinsfile

```
@@ -734,6 +734,8 @@ pipeline {
                         runTest stashes: [ 'CentOS-tests', 'CentOS-install', 'CentOS-build-vars' ],
                                 script: '''export PDSH_SSH_ARGS_APPEND="-i ci_key"
                                            # JENKINS-52781 tar function is breaking symlinks
+					   rm -rf $DAOS_BASE/test_results
+					   mkdir -p $DAOS_BASE/test_results
```

`$DAOS_BASE` has no value here.
```suggestion
					   mkdir test_results
```